### PR TITLE
WPS1 support

### PR DESCRIPTION
**Pull request by fmigneault**

### CODE REVIEW

File: weaver/tests/functional/application-packages/WPS1/DeployProcess_CatalogSearch.json [hunk 1]
```diff
@@ -38,8 +44,8 @@
     },
     "executionUnit": [
         {
-            "href": "catalog_search.cwl"
+            "href": "weaver/tests/functional/application-packages/WPS1/catalog_search.cwl"
         }
     ],
-    "deploymentProfileName": "http://www.opengis.net/profiles/eoc/workflow"
+    "deploymentProfileName": "http://www.opengis.net/profiles/eoc/wpsApplication"
```
Comment (dbyrns) [inline weaver/tests/functional/application-packages/WPS1/DeployProcess_CatalogSearch.json]: I know that the only options that we have right are "workflow" and "dockerizedApplication", so wpsApplication seems to make sense... But are we actually using that value?

File: weaver/wps_restapi/processes/processes.py [hunk 2]
```diff
@@ -518,7 +520,9 @@ def add_local_process(request):
             raise HTTPUnprocessableEntity("Invalid parameter 'processDescription.process.owsContext.offering.content'.")
         package = None
         reference = content.get('href')
-    elif deployment_profile_name.endswith('workflow') or deployment_profile_name.endswith('application'):
+    elif deployment_profile_name:
+        if not any(deployment_profile_name.endswith(typ) for typ in [PROCESS_APPLICATION, PROCESS_WORKFLOW]):
```
Comment (dbyrns) [inline weaver/wps_restapi/processes/processes.py]: I guess that I found the answer myself... So Dockerized and WpsApplication both end with application (but endswith isn't case sensitive?)

File: weaver/tests/functional/application-packages/WPS1/DeployProcess_CatalogSearch.json (hunk 1 above)
Comment (fmigneault) [reply weaver/tests/functional/application-packages/WPS1/DeployProcess_CatalogSearch.json]: yes, see your answer ;)

File: weaver/wps_restapi/processes/processes.py (hunk 2 above)
Comment (fmigneault) [reply weaver/wps_restapi/processes/processes.py]: there's a lower a few lines above

**Pull request merged**